From 6768c8efd31a8e1888fccf755269a8c1c5ceabcb Mon Sep 17 00:00:00 2001
From: Mao Minkai <maominkai@wxiat.com>
Date: Tue, 6 Sep 2022 16:39:05 +0800
Subject: [PATCH] sw64: enable DEBUG_BUGVERBOSE by default
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 6768c8efd31a8e1888fccf755269a8c1c5ceabcb
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I5XTGY

--------------------------------

Enable DEBUG_BUGVERBOSE by default to make debug easier.

Signed-off-by: Mao Minkai <maominkai@wxiat.com>
Reviewed-by: He Sheng <hesheng@wxiat.com>
Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/sw_64/Kconfig b/arch/sw_64/Kconfig
index 7174b6218539..17b48c7e5fc1 100644
--- a/arch/sw_64/Kconfig
+++ b/arch/sw_64/Kconfig
@@ -68,6 +68,7 @@ config SW64
 	select HAVE_ARCH_TRANSPARENT_HUGEPAGE
 	select HAVE_ASM_MODVERSIONS
 	select HAVE_C_RECORDMCOUNT
+	select HAVE_DEBUG_BUGVERBOSE
 	select HAVE_DYNAMIC_FTRACE
 	select HAVE_EBPF_JIT
 	select HAVE_FAST_GUP
-- 
2.33.0

